python/fastquant/strategies.py

```diff
@@ -18,6 +18,7 @@
 from collections.abc import Iterable
 import time
 from fastquant import get_bt_news
```

Comment:
I wonder if this should be `from .fastquant ...`

Reply:
nice find XD almost forgot